operations/bills.md

```diff
@@ -184,12 +223,26 @@ A bill is either a `Receipt` which means that it has been fully paid, or `Invoic
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `ItalianFiscalCode` | string  | optional | Italian fiscal code. |
+| `Id` | string  | optional | ID of the [Customer](customers.md#customer) to whom the bill was assigned. |
+| `Address` | [Bill address](#bill-address)  | optional | Address of the customer. |
+| `LegalIdentifiers` | Dictionary<string, string>  | optional | Legal Identifiers of the customer. |
+| `BillingCode` | string  | optional | ID of the Customer to whom the bill was assigned. |
+| `LastName` | string  | optional | Last name of the customer. |
+| `FirstName` | string  | optional | First name of the customer. |
+| `SecondLastName` | string  | optional | Second last name of the customer. |
+| `TitlePrefix` | [Title](customers.md#Title)  | optional | Title prefix of the customer. |
 
 #### Bill company data
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
+| `Id` | string  | optional | ID of the [Company](companies.md#company) to whom the bill was assigned. |
+| `Address` | [Bill address](#bill-address)  | optional | Address of the company. |
+| `LegalIdentifiers` | Dictionary<string, string>  | optional | Legal Identifiers of the company. |
+| `BillingCode` | string  | optional | ID of the company to whom the bill was assigned. |
```

Comment:
This can't be right, there is already a field for the company ID. What is BillingCode?

Reply:
Taken from our documentation. 'A unique code for Mews to list on invoices it sends to the company. The company can then easily recognize that the invoices are from your property.' I'll amend it to that, without the second sentence.